content/en/blog/2023/why-histograms/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Why Histograms?
+date: 2023-05-05
+author: '[Daniel Dyla](https://github.com/dyladan)'
+---
+
+<i>Originally posted to
+https://dyladan.me/histograms/2023/05/02/why-histograms/</i>
+
+A histogram is a multi-value counter which summarizes the distribution of one or
+more data points. For example, a histogram may have 3 counters which count the
+occurrences of negative, positive, and zero values respectively. Given a series
+of numbers, `3`, `-9`, `7`, `6`, `0`, and `-1`, the histogram would count `2`
+negative, `1` zero, and `3` positive values. A single histogram data point is
+most commonly represented as a bar chart.
```

Comment:
```suggestion
A histogram is a multi-value counter that summarizes the distribution of data points.
For example, a histogram may have 3 counters which count the
occurrences of negative, positive, and zero values respectively. Given a series
of numbers, `3`, `-9`, `7`, `6`, `0`, and `-1`, the histogram would count `2`
negative, `1` zero, and `3` positive values. A single histogram data point is
most commonly represented as a bar chart.
```
slight grammar change and changed "one or more" since people don't use histograms for 1 datapoint, even if it supports it